wsi/drm/drmdisplaymanager.cpp

```diff
@@ -446,10 +444,14 @@ void DrmDisplayManager::IgnoreUpdates() {
 }
 
 void DrmDisplayManager::setDrmMaster() {
-  int ret = drmSetMaster(fd_);
-  if (ret) {
-    ETRACE("Failed to call drmSetMaster : %s", PRINTERROR());
-  }
+    int ret = drmSetMaster(fd_);
+    while (ret) {
+        usleep(10000);
```

Comment:
Why usleep(10000)
Can we avoid to do so

Reply:
If the earlyEvs can drop the master successfully, it's no need for HWC to get the master again, but during my debug process, I saw due to the permission issue, the earlyEvs may drop the master failed, and the HWC failed to get the master, so the display screen will keep black forever.  
The master is critical for HWC, and this logic is a protection,  in case some error in earlyEvs or system, and HWC cannot get master currently, but may get it later, so it won't show black screen forever.